server/src/sql/data.sql

```diff
@@ -115,4 +115,4 @@ INSERT INTO project_categories(project_id, category_id)
            (3, 1), -- Project hack3 competes in ML
            (4, 1), -- Project hack4 competes in ML and Mobile
            (4, 2),
-           (5, 1); -- Project hack5 competes in ML
\ No newline at end of file
+           (5, 1); -- Project hack5 competes in ML
```

Comment:
probably remove this change in the commit, unless it's something w/ ts

Reply:
huh, i didn't add that line; perhaps got added when running it on my system. I'm gonna leave it